local-cli/runIOS/runIOS.js

```diff
@@ -18,6 +18,17 @@ const findMatchingSimulator = require('./findMatchingSimulator');
 const getBuildPath = function(configuration = 'Debug', appName, isDevice) {
   return `build/Build/Products/${configuration}-${isDevice ? 'iphoneos' : 'iphonesimulator'}/${appName}.app`;
 };
+const xcprettyAvailable = function() {
+  try {
+    child_process.execSync('xcpretty --version', {
+      stdio: [ 0, 'pipe', 'ignore', ]
+    });
+  } catch (error) {
+    console.log(error);
```

Comment:
I don't think we should log this, having this error print out could be confusing.